File: NgWebApp/src/app/features/location/models/address.ts
export interface Address {
  id: string;
  addressText: string;
  district: string;
  city: string;
  createdAt: string;
  updatedAt: string;
}

export type AddAddress = Omit<Address, 'id' | 'createdAt' | 'updatedAt'>;

File: BuyIt/LocationApi/Addresses/Address.cs
namespace LocationApi.Addresses;

public class Address : Contracts.Locations.LocationAddress
{
    public Guid Id { get; set; }
    public string AddressText { get; set; } = "";
    public string District { get; set; } = "";
    public string City { get; set; } = "";
    public DateTime CreatedAt { get; set; }
    public DateTime UpdatedAt { get; set; }
}
